lib/platform/user.js

```diff
@@ -205,6 +205,17 @@ const schema = schemaCreator.createSchema(
      * @inner
      */
     betaProgramParticipant: Boolean,
+    /**
+     * @name lastSeen
+     * @desc A timestamp with user last activity
+     * @type Date
+     * @memberof User
+     * @inner
+     */
+    lastSeen: {
+      type: Date,
+      default: Date.now,
```

Comment:
should be a call?

Reply:
Call will set the date only once, the moment when the schema is created.

Reply:
i saw it in that way in the mongoose docs